exercises/luhn/src/test/java/LuhnValidatorTest.java

```diff
@@ -66,11 +66,14 @@ public void testThatStringContainingPunctuationIsInvalid() {
         assertFalse(luhnValidator.isValid("055-444-285"));
     }
 
+    /* The following test diverges from the canonical test data. This is because the corresponding canonical test does
+     * not account for Java specific functions (such as Character.getNumericValue()), which can be part of incorrect yet
+     * passing implementations. For more detail, check out issue #972 here:
+     * (http://exercism.io/submissions/05a450216e9d4fc5965831a58760efe8).
```

Comment:
Is this definitely the right link? Do you not want to link to the issue: https://github.com/exercism/java/issues/972?